README.md

```diff
@@ -122,46 +122,46 @@ Each observer must have a publicly accessible HTTPS endpoint for receiving event
 
 ### Closing a campaign
 
-If an advertiser wants to close a campaign, they sign a new state, which distributes the remaining deposit: most of it goes to the advertiser's wallet, and a small part goes to the publisher platform as a cancellation fee.
+If an advertiser wants to close a campaign, they sign a new state, which distributes the remaining deposit: most of it goes back to the advertiser's wallet, and a small part goes to the publisher platform as a cancellation fee.
 
-The publisher-side platform recognizes this as an intention to close the campaign, and signs the state as well, therefore allowing the advertiser to withdraw their unspent funds. With this, the channel is considered exhausted, and it no longer represents any demand.
+The publisher-side platform recognizes this as an intention to close the campaign, and signs the state as well, therefore allowing the advertiser to withdraw their unspent funds. With this, the channel is considered exhausted and no longer represents any demand.
 
-While it is possible for a publisher-side platform to refuse to approve the state, they gain nothing from doing so: (1) the advertiser has decided to cancel the campaign, meaning they won't sign any new states with new payments to publishers anyway (2) after a channel is no longer valid, they still get their unspent deposit back and (3) the publisher-side platform gets compensated with a cancellation fee.
+While it is possible for a publisher-side platform to refuse to approve the state, they gain nothing from doing so: (1) the advertiser has decided to cancel the campaign, meaning they won't sign any new states with new payments to publishers anyway; (2) after a channel is no longer valid, they still get their unspent deposit back; and (3) the publisher-side platform gets compensated with a cancellation fee.
 
 ### Campaign health
 
-The campaign health is a publisher-specific concept which represents whether the advertiser is properly paying out after impression events.
+The campaign health is a publisher-specific concept that indicates whether the advertiser is properly paying out after impression events.
 
-Each publisher, with the help of the publisher-side platform, tracks the health status of each campaign they've ever interacted with. If a certain (configurable) threshold of non-paid impression events is reached, the campaign will be marked unhealthy, and the publisher will no longer pick it until the payment catches up.
+Each publisher, with the help of the publisher-side platform, tracks the health status of each campaign they've ever interacted with. If a certain (configurable) threshold of non-paid impression events is reached, the campaign will be marked unhealthy, and the publisher will no longer pick it until the outstanding payment is settled.
```

Comment:
false, not how it works; i literarily mean "catches up"; maybe needs some elaboration